cmd/commands/commands.go

```diff
@@ -1064,6 +1066,18 @@ var closeChannelCommand = cli.Command{
 				"success pobability of the negotiation if " +
 				"set higher",
 		},
+		cli.BoolFlag{
+			Name: "coop_close_with_htlcs",
```

Comment:
Think we want to name it as `no_wait` to stay in consistent with the RPC layer definition.

Reply:
yeah it's a bad name, but naming it differently here will make things worse, plus I don't think `no_wait` strictly equals `coop_close_with_htlcs`, as it's a flag introduced for compatibility reason.

Reply:
removed the flag